interview-questions.md

```diff
@@ -4,51 +4,51 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: Assgining the object keys to a variable 
 
-  Researched answer:
+  Researched answer :allows us to extract data from arrays, objects, maps and sets
 
 
 
 2. What are the three arguments that .map() can accept?
 
-  Your answer:
+  Your answer: value, index, and i dont recall the third
 
-  Researched answer:
+  Researched answer: (currentVal,index,array)
 
 
 
 3. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: a class is a blueprint for objects , and objects are instances of a class
 
   Researched answer:
 
 
 
 4. What is the difference between a HTML div and a span?
  
-  Your answer:
+  Your answer: the div tag is usually used as containers, and span is used to contain specific elements or text within a div 
 
-  Researched answer:
+  Researched answer: div is a block element, where as span is an inline element. Divs should be used to wrap larger sections and span to wrap lines of text. 
 
 5. What is React.Fragment and why would you want to use it?
   
-  Your answer:
+  Your answer: Its used to return multiple elements
   
-  Researched answer:
+  Researched answer:A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.
 
 6. What are three options for creating responsive design?
 
-  Your answer:
+  Your answer: 
```

Comment:
great job with all of the other answers! spot on. might want to do some more research on this one though ^^